[TASK] Update phpdomain/guides et all

linawolf opened the pull request:
phpdomain/guides was breaking by the renaming of interfaces.

Added a rule to remove unused imports, didnt want to do it by hand ;)

linawolf committed:
[TASK] Update phpdomain/guides et all

phpdomain/guides was breaking by the renaming of interfaces.

Added a rule to remove unused imports, didnt want to do it by hand ;)

brotkrueml commented:
@linawolf Rector complains. Additionally, I don't see a changed composer.lock file. Maybe this is the root cause ;-)

linawolf commented:
We do not have the composer.lock under version control...

brotkrueml commented:
Aah, sorry, it's domain, not guides...

linawolf committed:
[TASK] Update phpdomain/guides et all

phpdomain/guides was breaking by the renaming of interfaces.

require "rector/rector": "^0.19" as earlier versions fail with phpstan

Added a rule to remove unused imports, didnt want to do it by hand ;)

linawolf commented:
Tests work now, please have a look @brotkrueml
